follower-read.md

```diff
@@ -36,6 +36,7 @@ This variable is used to set the data read mode expected by the current session.
 - When the value of `tidb_replica_read` is set to `leader` or an empty string, TiDB maintains its original behavior and sends all read operations to the leader replica to perform.
 - When the value of `tidb_replica_read` is set to `follower`, TiDB selects a follower replica of the Region to perform all read operations.
 - When the value of `tidb_replica_read` is set to `leader-and-follower`, TiDB can select any replicas to perform read operations.
+- When the value of `tidb_replica_read` is set to `leader-and-follower`, read requests are load-balanced between the leader and followers.
```

Comment:
```suggestion
```
PLEASE pay attention to the context. @TomShawn line 38 already covers your new change.